### PR TITLE
config: mtl: add smart amp test module config

### PR DESCRIPTION
**Pull request by aiChaoSONG**
This patch adds smart amp test module config for mtl.

Signed-off-by: Chao Song <chao.song@linux.intel.com>

### CODE REVIEW

**Review by lgirdwood — DISMISSED** [review]

**Comment by lgirdwood** [comment]
@bardliao  @RanderWang pls review

File: config/mtl.toml [hunk 1]
```diff
@@ -393,3 +393,23 @@ count = 15
...
+
+	# smart amp test module config
+	[[module.entry]]
+	name = "SMATEST"
+	uuid = "167A961E-8AE4-11EA-89F1-000C29CE1635"
+	affinity_mask = "0x1"
+	instance_count = "1"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "9"
```
Comment (bardliao) [inline config/mtl.toml:405]: The module_type is 8 in [tgl-cavs.toml](https://github.com/thesofproject/rimage/blob/main/config/tgl-cavs.toml#L278). Is it correct?

Comment (lgirdwood) [reply config/mtl.toml:405]: Lets align all TGL to use MTL module types.

Comment (RanderWang) [reply config/mtl.toml:405]: It should be 13 
````
enum mod_type {
        ebasefw = 0,
        emixin,
        emixout,
        ecopier,
        epeakvol,
        eupdwmix,
        emux,
        esrc,
        ewov,
        efx,
        eaec,
        ekpb,
        emicselect,
        efxf,   /*i.e.SmartAmp */
        eaudclass,
        efakecopier,
        eiodriver,
        ewhm,
        egdbstub,
        esensing,
        emax,
        einvalid = emax
} ;
````

Comment (aiChaoSONG) [reply config/mtl.toml]: updated the module type for smart amp test to efxf (13).  the module types used between mtl and tgl are not aligned, ex, GAIN, for tgl, module_type=5, for mtl module_type=9, I will send another patch to make it aligned.

**Review by RanderWang — APPROVED** [review]

**Review by kv2019i — APPROVED** [review]

**Pull request merged**